app/mailers/user_mailer.rb

```diff
@@ -11,7 +11,7 @@ def welcome_user(user)
 	  mail(:subject => "Welcome to openSNP.org", :to => user.email)
   end
 
-  def genotyping_results(target_address,link,phenotype_name,variation)
+  def genotyping_results(target_address,link,phenotype_name,variation,user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used.